theano/sandbox/gpuarray/type.py

```diff
@@ -89,7 +122,8 @@ def filter_variable(self, other):
                             (str(other.type.broadcastable),
                              str(self.broadcastable)))
 
-        return theano.sandbox.gpuarray.basic_ops.gpu_from_host(other)
+        from .basic_ops import GpuFromHost
+        return GpuFromHost(self.context)(other)
```

Comment:
Can you confirm it is done? I think so, but the comments aren'c clear and the line number do not match anymore.